data/committees.js

```diff
@@ -57,6 +58,19 @@ export default [
 		],
 		introImage: '/images/committees/studio/studio_motif_applied.png',
 		mission: 'Our mission is to teach skills revolving around video game development such as computer science, game design and art in order to help usher students into the game development industry.',
+		whatInternsWillDo:
+		`Studio is mainly looking for interns to be in Unity Tutorial and Marketing/Engagement subcommittees this fall.
+		Marketing/Engagement- Studio hopes to bring interns to focus on community engagement, particularly in the form of leading socials during the year and figuring out strong ways to ensure member retention.
+		ACM Studio does not currently have a point person for socials, but we expect to have one (or two) soon, so interns can expect to work with our point person to assist with socials and other forms of community engagement.
+		Unity tutorial Subcommittees- ACM Studio has valued having helpers and headcount when hosting tutorials to help with various bugfixing issues to keep pace, while we also would like the extra manpower to continue
+		hosting tutorials in the winter quarter (which we haven't done in the past). Interns will not be expected to lead tutorials, but they will be expected to familiarize themselves with the tutorial content by attending the 
+		respective subcommittee meetings, making sure to ask the tutorial host questions before the event to ensure that the intern can help run the tutorial during the event. Prior skills in the Unity engine (which is Studio's
+		primary form of game programming) is heavily valued for this role but not entirely needed, as much of game development is a "learn as you go" type of style. Strong skills in C# (very similar to C++) and teaching 
+		skills are also appreciated, but these are all soft expectations.`,
+		timeCommitment: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Lectus nulla at volutpat diam ut. Elementum nisi quis eleifend quam adipiscing vitae.',
+		nextSteps: `An applicant fills out an officer interest form with various questions regarding skillset/reason for applying, then we schedule a 15-minute interview
```

Comment:
These keys may be less obvious for someone editing - do we want to nest this under a `internship` key?